spec/sidekiq/central_mail/submit_form4142_job_spec.rb

```diff
@@ -14,8 +18,22 @@
     # This is to ensure that the 2024 4142 template is
     # not used in tests unless explicitly enabled
     allow(Flipper).to receive(:enabled?).with(:disability_526_form4142_validate_schema).and_return(false)
+
+    # Stub out pdf methods as they are not needed for these tests and are cpu expensive
+    FileUtils.cp(fixture_pdf, test_pdf)
```

Comment:
@nfstern02 is `fixture_pdf` actually a pdf, just want to make sure. Thanks!

Reply:
Hi @stiehlrod, yes it is actually a pdf. Please see below.
<img width="1139" height="219" alt="Screenshot from 2025-11-05 14-15-30" src="https://github.com/user-attachments/assets/a7fafd55-8206-4092-b09a-8a74b9190ea2" />
<img width="1920" height="1160" alt="Screenshot from 2025-11-05 14-14-58" src="https://github.com/user-attachments/assets/a716b969-f5b6-417b-a06b-9eb622daf72b" />